.github/workflows/iroha2-dev-pr.yml

```
@@ -71,7 +71,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Build
-        run: ./scripts/build_wasm.sh
+        run: ./scripts/build_wasm.sh --profile=release
```

```suggestion
        run: ./scripts/build_wasm.sh
```

this should be the default